pylint/test/extensions/test_check_raise_docs.py

```diff
@@ -351,5 +351,44 @@ def my_func(self):
         with self.assertNoMessages():
             self.checker.visit_raise(raise_node)
 
+    def test_find_missing_sphinx_raises_infer_from_instance(self):
+        raise_node = astroid.extract_node('''
+        def my_func(self):
+            """This is a docstring.
+
+            :raises NameError: Never
+            """
+            my_exception = RuntimeError('hi')
+            raise my_exception #@
+            raise NameError('hi')
+        ''')
+        node = raise_node.frame()
+        with self.assertAddsMessages(
+            Message(
+                msg_id='missing-raises-doc',
+                node=node,
+                args=('RuntimeError', ))):
+            self.checker.visit_raise(raise_node)
+
+    def test_find_missing_sphinx_raises_infer_from_function(self):
```

Comment:
Also, add a test where we are raising an unknown exception and we don't complain about it. You can import it from a module which does not exist, I usually do ``from unknown import Unknown``